tests/test_cli.py

```diff
@@ -116,11 +116,25 @@ def test_render_output_build_path(testing_workdir, testing_metadata, capfd, capl
     assert error == ""
 
 
+def test_render_output_build_path_and_file(testing_workdir, testing_metadata, capfd, caplog):
+    api.output_yaml(testing_metadata, 'meta.yaml')
+    rendered_filename = 'out.yaml'
+    args = ['--output', '--file', rendered_filename, testing_workdir]
+    main_render.execute(args)
+    test_path = os.path.join(sys.prefix, "conda-bld", testing_metadata.config.host_subdir,
+                             "test_render_output_build_path_and_file-1.0-1.tar.bz2")
+    output, error = capfd.readouterr()
+    assert output.rstrip() == test_path, error
+    assert error == ""
+    rendered_meta = yaml.safe_load(open(rendered_filename, 'r'))
```

Comment:
Just another thing here (sorry for not spotted it earlier)
It would be better to use a context manager here with the ``open``. Something like:
```suggestion
    with open(rendered_filename, 'r') as rendered_yaml:
        rendered_meta = yaml.safe_load(rendered_yaml)
```

Reply:
>It would be better to use a context manager here with the `open`

OK, I've changed all the tests in the file not to use `open()` without a context manager.